digitalpy/core/zmanager/impl/default_routing_worker.py

```diff
@@ -212,15 +214,33 @@
         terminate = len(nextActionKey) == 0 or actionKey == nextActionKey
         if terminate:
             # stop processing
-            print("sending on %s", response_topic)
-            self.send_response(response, response_topic)
+            self.send_response(response, protocol)
             return
         
         self.process_next_request(controllerClass=controllerClass,response=response)
 
-        self.formatter.serialize(response)
+        self.send_response(response, protocol=protocol)
 
-        self.send_response(response, protocol=response_topic)
+    def get_response_topic(self, response: Response, protocol: str) -> List[bytes]:
+        """get the topic to which the response is to be sent
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D403 First word of the first line should be properly capitalized